scrapinghub/client/jobs.py

```diff
@@ -155,38 +239,40 @@ def schedule(self, spidername=None, **params):
             raise
         return Job(self._client, response['jobid'])
 
-    def get(self, jobkey):
-        """Get a Job with a given jobkey.
+    def get(self, job_key):
+        """Get a Job with a given job_key.
 
-        :param jobkey: a string job key.
+        :param job_key: a string job key.
 
-        jobkey's project component should match the project used to get
-        :class:`Jobs` instance, and jobkey's spider component should match
+        job_key's project component should match the project used to get
+        :class:`Jobs` instance, and job_key's spider component should match
         the spider (if :attr:`Spider.jobs` was used).
 
         :return: :class:`Job` object.
-        :rtype: scrapinghub.client.Job.
+        :rtype: scrapinghub.client.jobs.Job.
 
         Usage::
 
             >>> job = project.jobs.get('123/1/2')
             >>> job.key
             '123/1/2'
         """
-        jobkey = parse_job_key(jobkey)
-        if jobkey.projectid != self.projectid:
+        job_key = parse_job_key(job_key)
+        if job_key.project_id != self.project_id:
             raise ValueError('Please use same project id')
-        if self.spider and jobkey.spiderid != self.spider._id:
+        if self.spider and job_key.spider_id != self.spider._id:
             raise ValueError('Please use same spider id')
-        return Job(self._client, str(jobkey))
+        return Job(self._client, str(job_key))
 
-    def summary(self, _queuename=None, **params):
+    # FIXME there must me spider_name instead of spider_id for consistency
```

Comment:
`spider_name` or `spider`? list takes `spider`, schedule takes `spider_name` :) or maybe above methods should be filtered by `spider_id` for consistency with summary and iter_last?